doc/tutorial/extending_theano.txt

```diff
@@ -64,6 +64,13 @@ possibilities you may encounter or need.  For that refer to
         # Properties attribute
         __props__ = ()
 
+        #itypes and otypes attributes are
+        #compulsory if make_node method is not defined.
+        #They're the type of input and output respectively
+        itypes = []
+        otypes = []
```

Comment:
Make them default to None, to make it more clear that they aren't defined. There is some cases where you could have one op with no inputs and some outputs. Who know if we allow ops with no outputs in the futur (they could have have side effect for example)